cmake/iree_aie_xrt.cmake

```diff
@@ -0,0 +1,206 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+if(TARGET iree-aie-xclbinutil)
+  return()
+endif()
+
+include(${CMAKE_CURRENT_LIST_DIR}/iree_aie_utils.cmake)
```

Comment:
I actually do have that in our `iree_compiler_plugin.cmake` and `iree_runtime_plugin.cmake` (which is then how these get picked up elsewhere obv) and so actually I think I could just do `include(iree_aie_utils)` straight up? Reason I didn't is because I thought here it's better to be more explicit and also not give the impressio that the individual tools cmake (`xrt` `bootgen`, etc) should be included/used in each other.

Reply:
I don't have a strong opinion. Being explicit is fine.